dv/riscv_compliance/README.md

```diff
@@ -19,13 +19,14 @@ How to run RISC-V Compliance on Ibex
    - Verilator
    - fusesoc
    - srecord (for `srec_cat`)
-   - A RV32 compiler
+   - A RV32 compiler    # Toolchain provided with ibex will work
 
    On Ubuntu/Debian, install the required tools like this:
 
    ```sh
    sudo apt-get install srecord python3-pip
    pip3 install --user -U fusesoc
+   sudo apt install libelf-dev
```

Comment:
Can you fold this line into the apt-get line two lines above?

Reply:
Sure